portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase

```diff
@@ -2488,4 +2488,107 @@ definition {
 		FormFields.viewTextLabel(fieldLabel = "Text");
 	}
 
+	@description = "Case 1: Validating a String"
+	@priority = 5
+	test ValidateConfigurationStringAtControlPanel {
+		ApplicationsMenuHelper.openApplicationsMenu();
+
+		ApplicationsMenu.gotoPanel(panel = "Control Panel");
+
+		AssertElementPresent(
+			locator1 = "//h2[@id='nav_control_panel.configuration']");
+	}
+
+	@description = "Case 2: Writing tests in Forms"
+	@priority = 5
+	test WriteNewTestsInForms {
+
+		FormsAdmin.addForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.editDescription(formDescription = "Form 1 Description");
+
+		Form.editPageTitle(
+			formPageName = "Page 1",
+			formTitleNumber = 1);
+
+		Form.editPageDescription(
+			formPageDescriptionName = "Page 1 Description");
+
+		Form.gotoAddField(
+			fieldPositionNumber = 0,
+			fieldType = "Text");
+
+		Form.editFieldLabelMetal(fieldValue = "Full Name");
+
+		Form.closeSidebar();
+
+		Form.gotoAddField(
+			fieldPositionNumber = 1,
+			fieldType = "Numeric");
+
+		Form.editFieldLabelMetal(fieldValue = "Age");
+
+		Form.addTranslation(localizedLanguage = "Portuguese (Brazil)");
+
+		Click(
+            key_name = "Full Name",
+            locator1 = "FormFields#FIELD_TARGET_LABEL");
+
+		Form.editFieldLabelMetal(fieldValue = "Nome Completo");
+
+		Click(
+            key_name = "Age",
+            locator1 = "FormFields#FIELD_TARGET_LABEL");
+
```

Comment:
Why are you using Click function? You must use macro already created, you are only allowed to use function like Click in some cases. In this exercise, you are not allowed to use these functions, you must use macros for Liferay Portal. Review and send it correctly.